passes/memory/memlib.cc

```diff
@@ -878,7 +878,7 @@ struct Parser {
 						}
 					}
 				}
-				var.clk_en = find_single_cap(pdef.clken, cram.options, portopts, "clken");
+				var.clk_en = find_single_cap(pdef.clken, cram.options, portopts, "clken") != nullptr;
```

Comment:
I'm morbidly curious why this change is required 🤔 Is something about how `find_single_cap` is declared interfering with the implicit conversion to bool?

Reply:
The actual fix is assigning a default value to `PortVariant::clk_en`, this change was incidental while investigating but it made sense to me to leave it in since it helps with readability.